test/system/use_worker_pool_tests.rb

```diff
@@ -1,44 +1,48 @@
 require 'assert'
 require 'dat-worker-pool'
 
+require 'dat-worker-pool/worker'
+
 class DatWorkerPool
 
   class SystemTests < Assert::Context
     desc "DatWorkerPool"
-    subject{ @work_pool }
+    subject{ @worker_pool }
 
   end
 
   class UseWorkerPoolTests < SystemTests
     setup do
-      @mutex = Mutex.new
-      @results = []
-      @work_pool = DatWorkerPool.new({
-        :num_workers  => 2,
-        :logger       => TEST_LOGGER,
-        :do_work_proc => proc do |work|
-          @mutex.synchronize{ @results << (work * 100) }
-        end
+      @worker_class = Class.new do
+        include DatWorkerPool::Worker
+
```

Comment:
@jcredding I'm ok if you remove this empty line since it is just a test worker class - it is short enough to get what you are setting up.